content/for-developers/tutorials/inner-loop/expose-metrics/expose-metrics.md

```diff
@@ -57,3 +57,28 @@ Now, let's add the `ServiceMonitor` for our `stakater-nordmart-review-api` appli
 Voila, you successfully exposed metrics for your application!
 
 Move on to the next tutorial to see how to trigger alerts for your application.
+
+## Troubleshooting
+
+If something doesn't work here are some commonly useful things to check:
+
+- Start by doing curl on localhost from within the pod with the metrics path to make sure that metrics are exposed.
+
+```sh
+oc get pods -n namespace
+oc debug pod/podname-from-above
+curl localhost:port/path
+```
+
+- Verify everything in spec in the ServiceMonitor.
+    - Verify that endpoints port matches with the Service spec.ports.name .
+    - Verify that namespaceSelector matchnames matches with the Service metadata.namespace .
```

Comment:
@Andtalath can we give example of matches? Both should have same number of selectors or can be different on any side? Any examples we can add for newbies? In what cases it won't work?

Reply:
We could. Best way is probably a couple of scrubbed manifests.
However, the issue then becomes the level of specifity to document.

Reply:
Outside of conversation we agreed that providing examples is somewhat superflous in this context.